frontend/__tests__/components/language-switcher.test.tsx

```diff
@@ -0,0 +1,46 @@
+import { render, screen, waitFor } from '@testing-library/react';
+
+import { createRemixStub } from '@remix-run/testing';
+
+import { axe, toHaveNoViolations } from 'jest-axe';
```

Comment:
```suggestion
import { axe, toHaveNoViolations } from 'jest-axe';
import { describe, expect, it } from 'vitest';
```

We want to ensure we're using vitest everywhere instead of jest. The test still works with this change.

Reply:
I believe you're suggesting adding an explicit import from `vitest`?  If so, added.